sql/wsrep_mysqld.cc

```diff
@@ -2004,6 +2004,27 @@ static void wsrep_RSU_end(THD *thd)
   thd->variables.wsrep_on= 1;
 }
 
+
+static bool global_read_lock_set(THD *thd)
+{
+  MDL_key key;
+  THD *tmp;
+  bool ftwrl1_lock= false, ftwrl2_lock= false;
+  if (thd->global_read_lock.is_acquired()) return 1;
```

Comment:
please change "return 1" to "return true"

Reply:
Changed